css/style.css

```diff
@@ -0,0 +1,75 @@
+
```

Comment:
- folder structure has to be followed
- css has to be divided into many parts just like i have done in my POC demo's
- use BEM convention for css